### PR TITLE
E2e/feature/ri 4806 sso feature flag

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/tests/regression/database/cloud-sso.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+const pathes = {
+    defaultRemote: path.join('.', 'test-data', 'features-configs', 'insights-default-remote.json'),
+    dockerConfig: path.join('.', 'test-data', 'features-configs', 'sso-docker-build.json'),
+    electronConfig: path.join('.', 'test-data', 'features-configs', 'sso-electron-build.json')
+};
+
+fixture `Cloud SSO`
+    .meta({ type: 'regression', rte: rte.standalone })
+    .page(commonUrl)
+    .beforeEach(async() => {
```
Comment (mariasergeenko) [inline tests/e2e/tests/regression/database/cloud-sso.e2e.ts:25]: if you reload before function. is beforeEach need here?

File: tests/e2e/tests/regression/database/cloud-sso.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        await databaseAPIRequests.deleteAllDatabasesApi();
+        await refreshFeaturesTestData();
```
Comment (mariasergeenko) [inline tests/e2e/tests/regression/database/cloud-sso.e2e.ts:27]: shouldn't we add addDataBase function here?

Comment (vlad-dargel) [reply tests/e2e/tests/regression/database/cloud-sso.e2e.ts:27]: no

File: tests/e2e/tests/regression/database/cloud-sso.e2e.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/tests/regression/database/cloud-sso.e2e.ts:25]: yes, agree, deleted before hook in the first test

**Review by mariasergeenko — APPROVED** [review]

**Pull request merged**